lte/cloud/go/services/lte/servicers/builder_servicer_test.go

```diff
@@ -161,6 +162,9 @@ func TestBuilder_Build(t *testing.T) {
 			LogLevel:          protos.LogLevel_INFO,
 			DhcpServerEnabled: true,
 		},
+		"li-agentd": &lte_mconfig.LIAgentD{
+			LogLevel: protos.LogLevel_INFO,
+		},
```

Comment:
can you add some actual test that goes through your code. 
It will shield it from future modifications

Reply:
I will figure out a way to test this part of code in a separate PR as it requires some work to mock configurator LoadAllEntitiesOfType call.